src/libsync/propagatedownload.cpp

```diff
@@ -639,6 +644,27 @@ void PropagateDownloadFile::slotGetFinished()
         return;
     }
 
+    // Did the file come with conflict headers? If so, store them now!
+    // If we download conflict files but the server doesn't send conflict
+    // headers, the record will be established by SyncEngine::conflictRecordMaintenance.
+    // (we can't reliably determine the file id of the base file here,
+    // it might still be downloaded in a parallel job and not exist in
+    // the database yet!)
+    if (propagator()->account()->capabilities().uploadConflictFiles()
```

Comment:
We should store in the database even if the capability is not set, i'd say

Reply:
Why would a server without the uploadConflictFiles capability send us a file that includes conflict headers? It'd be odd: we'd download the file and store it in the db, but due to the exclude pattern I don't think we'd upload any local changes.

Reply:
Because we change the option locally with the env variable.
Or the files were uploaded before the server changed its config.